ui/app/pages/send/send-content/send-asset-row/send-asset-row.component.js

```diff
@@ -121,7 +127,8 @@ export default class SendAssetRow extends Component {
         <div className="send-v2__asset-dropdown__asset-icon">
           <Identicon
             diameter={36}
-            address={nativeCurrency === ETH ? undefined : nativeCurrency}
+            image={nativeCurrencyImage}
+            address={nativeCurrency}
```

Comment:
Passing in the native currency as the "address" seems wrong :thinking: I see that we were already doing this for non-ETH native currencies though, for some reason :confused:

Reply:
The effect of this, and perhaps this was the intent, is to have a jazzicon display when there is a custom symbol for the custom network. Like this:
![Screenshot from 2021-03-31 20-26-23](https://user-images.githubusercontent.com/7499938/113222321-49a58780-9261-11eb-9654-dafd9839ee04.png)

If no symbol is provided, it looks like this.

![Screenshot from 2021-03-31 20-27-00](https://user-images.githubusercontent.com/7499938/113222310-45796a00-9261-11eb-8a9f-382baf1c9809.png)

I agree that we should not do this, and instead should just show the new default of Identicon: `<div style={getStyles(diameter)}></div>;`

Now, the only problem with changing this is if some users have become acustomed to seeing the jazzicon for their particular custom network... so I think it'd be best not to break that in this PR.

Reply:
Yeesh. So we're treating this as an address without validating that it's even remotely similar to an address. That's unfortunate.

Reminds me of this bug: https://github.com/MetaMask/metamask-extension/issues/8252.

Leaving this alone for this PR seems sensible to me. I'm not sure we should be concerned about users being accustomed to this icon though. It seems very broken and confusing to me. Particularly because we use jazzicons for people, and we don't want tokens mistaken for people. Maybe we can discuss it in a future standup.